backends/iptables/iptables.go

```diff
@@ -256,7 +257,7 @@ func (t *iptables) sync() {
 }
 
 func (t *iptables) createServiceSpecificChains(svcInfo *serviceInfo, activeNATChains map[util.Chain]bool,
-	existingNATChains map[util.Chain][]byte, allEndpoints *endpointsInfoByName) ([]*string, *[]util.Chain, *[]util.Chain, map[string]map[string]int32) {
+	existingNATChains map[util.Chain][]byte, allEndpoints *endpointsInfoByName) ([]*string, *[]util.Chain, *[]util.Chain, map[string]int32) {
```

Comment:
oh so were replacing 

```
str0 -> 
   str1 -> int1
   str2 -> int2
```

w/ 
```
str0 -> int0
str1 -> int1 
```

why ? sorry am reloading my kpng brain been focused on downstream for a few months